### PR TITLE
snap: add `snap debug state --is-seeded` helper

### PR DESCRIPTION
**Pull request by mvo5**
This commit allows to get the seeded status via the snap debug
command. This will be used in the snapd postinst to double
check if seeding worked to avoid an expensive seed check if
it's not needed.

### CODE REVIEW

File: cmd/snap/cmd_debug_state.go [hunk 1]
```diff
@@ -78,11 +80,12 @@ func init() {
...
-		"task":    i18n.G("ID of the task to inspect"),
-		"dot":     i18n.G("Dot (graphviz) output"),
-		"no-hold": i18n.G("Omit tasks in 'Hold' state in the change output"),
-		"changes": i18n.G("List all changes"),
+		"change":    i18n.G("ID of the change to inspect"),
+		"task":      i18n.G("ID of the task to inspect"),
+		"dot":       i18n.G("Dot (graphviz) output"),
+		"no-hold":   i18n.G("Omit tasks in 'Hold' state in the change output"),
+		"changes":   i18n.G("List all changes"),
+		"is-seeded": i18n.G("Output is seeded successfully"),
```
Comment (zyga) [inline cmd/snap/cmd_debug_state.go:88]: Output? I would expect something like `System is seeded ...`

File: cmd/snap/cmd_debug_state_test.go [hunk 2]
```diff
@@ -34,7 +34,8 @@ var stateJSON = []byte(`
 	"last-change-id": 2,
 
 	"data": {
-		"snaps": {}
+		"snaps": {},
+                "seeded": true
```
Comment (zyga) [inline cmd/snap/cmd_debug_state_test.go:38]: Tabs/spaces issue?

File: cmd/snap/cmd_debug_state.go (hunk 1 above)
Comment (mvo5) [reply cmd/snap/cmd_debug_state.go]: I updated the text now.

File: cmd/snap/cmd_debug_state_test.go (hunk 2 above)
Comment (mvo5) [reply cmd/snap/cmd_debug_state_test.go:38]: should be fixed now

**Comment by mvo5** [comment]
For context see https://github.com/snapcore/snapd/pull/8382 how it is used

File: cmd/snap/cmd_debug_state.go (hunk 1 above)
Comment (zyga) [reply cmd/snap/cmd_debug_state.go]: Can you check, it looks like the same text is used.

Comment (mvo5) [reply cmd/snap/cmd_debug_state.go]: Can you please reload?

**Pull request merged**